### ISSUE TITLE
Doesn't work with foundry vtt 0.8.9

### ISSUE THREAD

**Issue by cyco-polo** [relick/FoundryVTT-Right-Click-Enabler#2]
My screen gets blocked and i loose acces the bottom side bar and right side bar
![Capture d’écran 2021-08-21 224828](https://user-images.githubusercontent.com/80625501/130335109-4d127b9b-ecbd-481b-8e1d-4b91e24ee301.png)

**Comment by relick** [comment]
Could you try out the new version of right-click-enabler (0.0.3) and report back if it works, please?

**Comment by ryankhart** [comment]
I've tested in Microsoft Edge with right-click-enabler (0.0.3) and Foundry VTT (0.8.9) and it works for me. I found this because I initially thought it didn't work, but that's because I was new and I didn't know that I needed to enable modules after installing them before they go into effect from the "Manage Modules" button in the "Game Settings" tab.

By the way, before anyone mocks me for using Microsoft Edge, I use it because it is the only browser I know that has built-in text-to-speech for almost any content on the web (except for any website that hijacks the right-click context menu). It even has an option for premium voices for free that ordinarily, you'd have to pay a subscription for.
